semantic/src/site/globals/site.overrides

```diff
@@ -1555,6 +1555,10 @@ a:hover {
   margin-bottom: 20px;
 }
 
+.stage-label {
```

Comment:
This is all that's required to fix #1504. Just prevents any wrapping within the label.